services/config.tsx

```diff
@@ -0,0 +1,25 @@
+const CONFIG_PATH = 'https://raw.githubusercontent.com/cowprotocol/cow-fi/configuration/config/'
+
+export interface CowStats {
+  totalTrades: number, // https://dune.com/queries/1034337
+  surplus: { // https://dune.com/queries/270604
+    reasonable: number,
+    unusual: number
+  },
+  lastModified: Date
+}
+
+type CowStatsConfig = Omit<CowStats, 'lastModified'> & { lastModified: string }
+
+async function getFromConfig<T>(configFilePath: string): Promise<T> {
+  const response = await fetch(CONFIG_PATH + `${configFilePath}`)
+  return await response.json()
+}
```

Comment:
What about some error handling in case this call/json decoding fails?

Reply:
I want it to fail if there's an error. This way, it will use the old value